staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -310,52 +308,6 @@ var (
 			},
 		},
 	)
-	// We add an explicit rule for endpoint-controller with high precedence
-	// to ensure that those calls won't get caught by the following
-	// <workload-leader-election> flow-schema.
-	//
-	// TODO(#80289): Get rid of this rule once we get rid of support for
-	//   using endpoints and configmaps objects for leader election.
-	SuggestedFlowSchemaEndpointsController = newFlowSchema(
-		"endpoint-controller", "workload-high", 150,
-		flowcontrol.FlowDistinguisherMethodByUserType,
-		flowcontrol.PolicyRulesWithSubjects{
-			Subjects: append(
-				users(user.KubeControllerManager),
-				kubeSystemServiceAccount("endpoint-controller", "endpointslicemirroring-controller")...),
-			ResourceRules: []flowcontrol.ResourcePolicyRule{
-				resourceRule(
-					[]string{"get", "create", "update"},
-					[]string{corev1.GroupName},
-					[]string{"endpoints"},
-					[]string{flowcontrol.NamespaceEvery},
-					false),
-			},
-		},
-	)
-	// TODO(#80289): Get rid of this rule once we get rid of support for
-	//   using endpoints and configmaps objects for leader election.
-	SuggestedFlowSchemaWorkloadLeaderElection = newFlowSchema(
-		"workload-leader-election", "leader-election", 200,
-		flowcontrol.FlowDistinguisherMethodByUserType,
-		flowcontrol.PolicyRulesWithSubjects{
-			Subjects: kubeSystemServiceAccount(flowcontrol.NameAll),
-			ResourceRules: []flowcontrol.ResourcePolicyRule{
-				resourceRule(
-					[]string{"get", "create", "update"},
-					[]string{corev1.GroupName},
-					[]string{"endpoints", "configmaps"},
-					[]string{flowcontrol.NamespaceEvery},
-					false),
```

Comment:
only delete this stanza, we still need the flowcontrol on the leases objects

Reply:
Requests from 1. and 2. would be matched by the `kube-controller-manager` FlowSchema which classifies these requests to the `workload-high` Priority Level - so from that POV, dropping those two FlowSchemas is a no-op.

Wrt. 3., such requests would be matched by the `kube-system-service-accounts` FlowSchema - resulting in requests ending up in the `workload-high` Priority Level (instead of `leader-election`).